script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2_1.txt

```diff
@@ -67,6 +68,25 @@ The Tohno family was a wealthy family and also shareholders of several companies
 An anachronistic building.
 The life of the upper class, respecting civility and history.
 Such a life in the mansion seemed boring to my childish mind.
+That's why, when I was told that I would be left at the Arima house, I accepted without much resistance.
+<Page20>
+I think it was for the best, in the end.
+I got along well with the people of the Arima family, and I treated my stepmother Keiko and stepfather Fumiomi as if they were my real parents.
+Originally longing for a normal family, Tohno Shiki lived with the Arima family as though he were their actual son.
+<Page21>
+I don't think there was any regret in that.
+The only thing I regret is that I left behind my younger sister, Akiha.
+<Page22>
+Akiha must have hated me... Tohno Shiki.
+Or rather, she has every right to hate me.
+She was left alone in that huge mansion, living with my hard-headed father.
+It's easy to imagine how Akiha feels about me running away from her.
+<Page18>
+The Tohno family was a wealthy family and also shareholders of several companies. My father, Tohno Makihisa, was apparently a mediocre businessman, but the assets he inherited from his grandfather were strong enough to keep the company running smoothly.
+<Page19>
+An anachronistic building.
+The life of the upper class, respecting civility and history.
+Such a life in the mansion seemed boring to my childish mind.
```

Comment:
uhhh what's goin on here

Reply:
I'm a dummy and copy-pasted wrong when trying to make it match the other file. It's fixed now.